crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -1149,6 +1202,9 @@ mod tests {
         .await
         .unwrap();
 
+        let faucet_clone = Arc::clone(&faucet);
+        let cloned = &mut Arc::try_unwrap(faucet_clone).unwrap();
```

Comment:
Additionally, to make sure `try_unwrap` is safe, you will also need to shut down the batch task before you do it (so that it also doesn't try to take a strong reference and cause this call to panic sometimes).

Reply:
Oh yes this did not work which is why its removed.